LayoutTests/fast/events/focus-keyboard-event-crash.html

```diff
@@ -0,0 +1,28 @@
+<script>
+(async () => {
+    globalThis.testRunner?.waitUntilDone();
+    globalThis.testRunner?.dumpAsText();
+
+    const link = document.createElement('link'),
+          section = document.createElement('section'),
+          dialog = document.createElement('dialog'),
+          a = document.createElement('a');
+
+    document.documentElement.prepend(link);
+    link.before(section);
+    section.append(dialog);
+    dialog.append(a);
+
+    history.back();
+    a.href = URL.createObjectURL(new MediaSource({ detachable: true }));
+
+    try {
+        await window.indexedDB.databases();
+        await frames.caches.has(`\u38ec\u0f09\u0f2d\u0989\u{1fc70}`);
+    } catch {}
+
+    dialog.showModal();
+    globalThis.testRunner?.notifyDone();
```

Comment:
Before this line, can we add something like `document.write('This test passes WebKit does not crash<br>PASS');` right before notifyDone so that there is some output text?
After that, you need to run the test with `run-webkit-tests --reset <path-to-test>` to update the expected result.

Reply:
I added this locally to test but the expected test file is not getting updated.